README.md

```diff
@@ -57,6 +57,7 @@ Awesome Artificial Intelligence, Machine Learning and Deep Learning as we learn
     - [Programming in Python](Programming-in-Python.md)
     - [More...](julia-python-and-r.md#more)
   - [JavaScript](README-details.md#javascript)
+  - [Swift for TensorFlow](https://www.tensorflow.org/swift/) - next generation platform for deep learning and differentiable programming
```

Comment:
You mean Tensorflow for Swift? Can you please fix it!

Reply:
I see the main link also says Swift for Tensorflow - although previous releases of Tensorflow were titled like on this page https://github.com/tensorflow/tensorflow/tree/master/tensorflow/java

Reply:
Yeah, they call it `Swift for TensorFlow` as it's not a TensorFlow API for Swift, it's a bigger project to build a next generation platform for deep learning and differentiable programming.